askama/src/filters/alloc.rs

```diff
@@ -776,6 +816,93 @@ fn flush_title(dest: &mut (impl fmt::Write + ?Sized), s: &str) -> fmt::Result {
     Ok(())
 }
 
+/// A prefix usable for indenting [prettified JSON data](super::json::json_pretty) and
+/// [`|indent`](indent)
+///
+/// ```
+/// # use askama::filters::AsIndent;
+/// assert_eq!(4.as_indent(), "    ");
+/// assert_eq!(" -> ".as_indent(), " -> ");
+/// ```
+pub trait AsIndent {
+    /// Borrow `self` as prefix to use.
+    fn as_indent(&self) -> &str;
+}
+
+impl AsIndent for str {
```

Comment:
`CowStr` I suppose too?

Reply:
`Cow<'_, T>` is implemented further down.